.github/workflows/tests.yml

```diff
@@ -55,14 +55,18 @@ jobs:
           pip install .
           pip install .[test]
 
-      - name: Static type checking with mypy
+      - name: Static type checking (mypy)
         run: |
           # remove '# type:' comments in version.py which mypy-3.8
           # stumbles over. for some reason, mypy cannot be
           # instructed to ignore 'version.py'
           sed -i "s/# type:.*//" odxtools/version.py
           python -m mypy .
 
+      - name: Lint code quality (ruff)
```

Comment:
What happens if ruff and yapf don't agree?

Reply:
a lot of fun ;).

If the disagreement cannot be fixed by "normal" means, the ruff check can be turned off globally in `pyproject.toml` (while doing this PR, this happened with the sorting of the import statement), or ruff can be instructed to ignore a certain error at a given location using `# noqa: $ERRNO` comments (there are some of these in this PR as well...)